style.css

```diff
@@ -404,3 +404,118 @@ nav #mylogo {
   justify-content: space-between;
   list-style: none;
 }
+
+.container {
+  display: flex;
+  width: 100%;
+  height: auto;
+  flex-direction: column;
+  border-top-left-radius: 80px;
+  padding: 115px 25px;
+  background-color: #6070ff;
+  padding-bottom: 40px;
+  position: relative;
+}
+
+.container::before {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 9rem;
+  background-image: url("./images/icon01.png"), url("./images/icon02.png");
+  background-size: 6rem, 2rem;
+  background-repeat: no-repeat;
+  background-position: 100% 99%, 21% 52%;
+  position: absolute;
+  right: 0%;
+  top: 1%;
+}
+
+.container::after {
+  content: "";
+  display: block;
+  width: 19rem;
+  height: 9rem;
+  background-image: url("./images/icon03.png");
+  background-size: 6rem;
+  background-repeat: no-repeat;
+  background-position: 100% 99%;
+  position: absolute;
+  right: 15%;
+  bottom: 1%;
+}
+
+.container-header {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  padding: 0;
+  margin: 0;
+  width: 801px;
+  height: 56px;
+  padding-left: 40px;
+}
+
+.container-text {
+  font-weight: 400;
+  font-size: 20px;
+  color: #ebebff;
+  line-height: 28px;
+  text-align: center;
+  width: 327px;
+  margin: 0;
+  margin-top: 12px;
+  font-style: normal;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  margin-top: 50px;
+}
+
+.contact-form input {
+  border-radius: 12px;
+  width: 327px;
+  height: 48px;
+  border: none;
+}
+
+.contact-form textarea {
+  height: 170px;
+  width: 327px;
+  border-radius: 12px;
+}
+
+.center-placeholder::-webkit-input-placeholder {
+  left: 16px;
+  right: 38px;
+  top: 15px;
+}
+
+.btn {
+  padding: 12px 16px;
+  width: 142px;
+  height: 48px;
+  border-radius: 8px;
+  border: none;
+  background: #fff;
+}
+
+.btn:hover {
+  transform: scale(1.3);
+  transition-duration: 0.3s;
+  background-color: #6070ff;
+  color: white;
+  border: 1px solid black;
+}
+
+.btn:active {
+  background-color: #2230d2;
+  color: #fff;
+}
```

Comment:
- [ ] Nice Design so far but something need to be improved for your project to look professional , I just noticed that your container didn't fit all the screen , your mobile version should be responsive , kindly fix this in the video I presented below :upside_down_face:  :point_down: 

https://user-images.githubusercontent.com/88894525/220473258-4795a002-532d-464c-b31a-9e76ef700145.mp4

Reply:
@Alfredbis29 this video has no sound .will you explain it  well?